packages/slack/lib/publish.js

```diff
@@ -11,11 +11,12 @@ if (SLACK_API_TOKEN) {
   console.warn('Posting to slack disabled: missing SLACK_API_TOKEN')
 }
 
-const publish = async (channel, content) => {
+const publish = async (channel, content, options = {}) => {
   if (webClient && channel) {
     await webClient.chat.postMessage({
       channel,
-      text: content
+      text: content,
+      ...options
```

Comment:
I would expect optional `options` not to overwrite first two arguments `channel`, `content`.

I'd go for: Spread options first, then add `channel`, `text`.

```js
webClient.chat.postMessage({ ...options, channel, text })
```